src/main/java/seedu/address/model/person/Person.java

```diff
@@ -133,6 +142,10 @@ public String toString() {
             builder.append("; Tags: ");
             tags.forEach(builder::append);
         }
+        Company company = getCompany();
+        if (company != null) {
+            builder.append(("; Company: ")).append(getCompany());
+        }
```

Comment:
Based on `getCompany()`, company will never be null (at best, `new Company(null)`)
It is fine to print it even if there is no company since a company with fullname `null` would have the string "No company", so NPE is avoided